client/client.go

```diff
@@ -175,19 +175,7 @@ func (c *Client) downloadRoot() error {
 	var s *data.Signed
 	var raw []byte
 	if download {
-		logrus.Debug("downloading new root")
-		raw, err = c.remote.GetMeta(role, size)
-		if err != nil {
-			return err
-		}
-		hash := sha256.Sum256(raw)
-		if expectedSha256 != nil && !bytes.Equal(hash[:], expectedSha256) {
```

Comment:
Thanks for taking a look. I believe you're right, when downloading the root specifically we may not have an expected sha256. I intentionally made it more restrictive, so I'll have to take a look at all the test cases because everything from TUF, through Notary, to Docker integration tests passed and looking at this again, I believe there should have been a failure somewhere.